src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const partnerSex = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+
+      {person.age && (
+        <p className="Person__age">I am { person.age }</p>
+      )}
+
+      {person.isMarried ? (
+        <p className="Person__partner">{person.partnerName} is my { partnerSex }</p>
+      ) : (
+          <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
```suggestion
        <p className="Person__partner">
          {person.isMarried ? `${person.partnerName} is my ${partnerSex}` : 'I am not married'}
        </p>
```
enough decide only which text will be rendered.